department-of-veterans-affairs/va.gov-cms
# VACMS-7161: Last updated by [revision user] is broken on /admin/content

- Updated the "Facilities", "Resources and support", "Bulk edit" and "Events" content views displays; moved the "Content revision: Revision user" field on top of the "Content: Changed (Updated)" field and set it to hidden. Then edited the "Content: Changed (Updated)" and with the "Rewrite results" put the fields together "{{ changed }} by {{ revision_uid }}".

## Description

closes #7161 

## Testing done

All test passed

## Screenshots

**Content:**
![VACMS-7161-content](https://user-images.githubusercontent.com/846871/145276340-9cc29500-89ba-4a44-b02b-7f10ffca79dd.png)

**Facilities:**
![VACMS-7161-facilities](https://user-images.githubusercontent.com/846871/145276379-8b605f70-a6db-4ab1-9d4b-a19f4a939970.png)

**Resources and support:**
![VACMS-7161-resources](https://user-images.githubusercontent.com/846871/145276409-f4bd3c37-572e-40f3-ac13-87075f327f8e.png)

**Bulk edit:**
![VACMS-7161-bulk](https://user-images.githubusercontent.com/846871/145276430-5b6ad40f-e17d-42b0-beb3-80bf6ae346a9.png)

**Events:**
![VACMS-7161-events](https://user-images.githubusercontent.com/846871/145276454-6f20da5d-805a-4b56-86d4-fa9b3893acdc.png)


## QA steps

As user administrator
1. Go to `/admin/content`
    - [x] Verify and confirm that the "Updated" column reads DATE - TIME by USER, e.g.: 11/22/2021 - 20:59 pm by alex.britt@va.gov
1. Go to `/admin/content/facilities` 
    - [x] Verify and confirm that the "Updated" column reads DATE - TIME by USER, e.g.: 11/22/2021 - 20:59 pm by alex.britt@va.gov 
1. Go to `/admin/content/resources-and-support`
    - [x] Verify and confirm that the "Updated" column reads DATE - TIME by USER, e.g.: 11/22/2021 - 20:59 pm by alex.britt@va.gov 
1. Go to `/admin/content/bulk`
    - [x] Verify and confirm that the "Updated" column reads DATE - TIME by USER, e.g.: 11/22/2021 - 20:59 pm by alex.britt@va.gov 
1. Go to `/admin/content/events`
    - [x] Verify and confirm that the "Updated" column reads DATE - TIME by USER, e.g.: 11/22/2021 - 20:59 pm by alex.britt@va.gov 

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
